config/routes.rb

```diff
@@ -1,76 +1,85 @@
-  root 'sufia/homepage#index'
-  curation_concerns_collections
-  curation_concerns_basic_routes
-  curation_concerns_embargo_management
-  concern :exportable, Blacklight::Routes::Exportable.new
+    resource :catalog, only: [:index], as: 'catalog', path: '/catalog', controller: 'catalog' do
+      concerns :searchable
+      concerns :range_searchable
 
-  # there might be a way to get curation_concerns routes to this for us,
-  # but don't know it, and this is easy enough and works. Make the viewer
-  # URL lead to ordinary show page, so JS can pick it up and launch viewer.
-  get '/concern/generic_works/:id/viewer/:filesetid(.:format)' => 'curation_concerns/generic_works#show'
-  get '/concern/parent/:parent_id/generic_works/:id/viewer/:filesetid(.:format)' => 'curation_concerns/generic_works#show'
+    end
 
+    devise_for :users
+    # https://github.com/plataformatec/devise/wiki/how-to:-change-the-default-sign_in-and-sign_out-routes
+    devise_scope :user do
+      get 'login', to: 'devise/sessions#new'
+    end
 
-  resources :solr_documents, only: [:show], path: '/catalog', controller: 'catalog' do
-    concerns :exportable
-  end
+    # make the contact form inaccessible since we're using mailto
+    get 'contact', to: redirect('/404')
 
-  resources :bookmarks do
-    concerns :exportable
+    resources :welcome, only: 'index'
+    root 'sufia/homepage#index'
+    curation_concerns_collections
+    curation_concerns_basic_routes
+    curation_concerns_embargo_management
+    concern :exportable, Blacklight::Routes::Exportable.new
 
-    collection do
-      delete 'clear'
+    # there might be a way to get curation_concerns routes to this for us,
+    # but don't know it, and this is easy enough and works. Make the viewer
+    # URL lead to ordinary show page, so JS can pick it up and launch viewer.
+    get '/concern/generic_works/:id/viewer/:filesetid(.:format)' => 'curation_concerns/generic_works#show'
+    get '/concern/parent/:parent_id/generic_works/:id/viewer/:filesetid(.:format)' => 'curation_concerns/generic_works#show'
+
+
+    resources :solr_documents, only: [:show], path: '/catalog', controller: 'catalog' do
+      concerns :exportable
     end
-  end
 
-  # local routes
-  get '/opac_data/:rec_num', to: 'opac_data#load_bib'
-  mount Hydra::RoleManagement::Engine => '/'
+    resources :bookmarks do
+      concerns :exportable
 
-  get '/focus/:id', to: 'synthetic_category#show', as: :synthetic_category
+      collection do
+        delete 'clear'
+      end
+    end
 
+    # local routes
+    get '/opac_data/:rec_num', to: 'opac_data#load_bib'
+    mount Hydra::RoleManagement::Engine => '/'
 
-  Hydra::BatchEdit.add_routes(self)
-  # Sufia should be mounted before curation concerns to give priority to its routes
-  mount Sufia::Engine => '/'
-  mount CurationConcerns::Engine, at: '/'
+    get '/focus/:id', to: 'synthetic_category#show', as: :synthetic_category
+
+
+    Hydra::BatchEdit.add_routes(self)
```

Comment:
unrelated to this PR, but this line is in the routes twice. I guess we should remove the later one.